ember-welcome-page/babel.config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "presets": ["@babel/preset-typescript"],
```

Comment:
Here, I diverged from the [line that `embroider-build/addon-blueprint` generates](https://github.com/embroider-build/addon-blueprint/blob/v1.3.0/files/__addonLocation__/babel.config.json#L2):

```json5
{
  "presets": [["@babel/preset-typescript"]],
  ...
}
```

The [Babel documentation](https://babeljs.io/docs/en/babel-preset-typescript#with-a-configuration-file-recommended) seemed to suggest that single brackets are enough. I'm not sure what the difference in syntax means.

@simonihmig If you have time, could you help me understand?